package org.example.factorymethod;

public class SMSNotification implements Notification {

    @Override
    public void send(String message) {
        System.out.println("sending sms: " + message);
    }
}
